Modules/makesetup

```diff
@@ -87,18 +87,6 @@ esac
 NL='\
 '
 
-# Setup to link with extra libraries when making shared extensions.
-# Currently, only Cygwin needs this baggage.
-case `uname -s` in
-CYGWIN*) if test $libdir = .
```

Comment:
I've removed this in favor of the "shared libraries must be linked with libpython" section in configure.ac, which sets the MODULE_LDFLAGS variable used below.